architecture.md

```diff
@@ -361,7 +359,7 @@ process representing this container process.
 Communication with the VM can be achieved by either `virtio-serial` or, if the host
 kernel is newer than v4.8, a virtual socket, `vsock` can be used. The default is `virtio-serial`.
 
-The VM will likely be running multiple container processes.  In the event `virtio-serial`
+The VM likely will be running multiple container processes.  In the event `virtio-serial`
```

Comment:
Heh, my head reads it better as the original way around - I think this is one for @klynnrif :-)